test/functional/test-standard-actions.js

```diff
@@ -23,36 +23,109 @@ import {setParentWindow} from '../../src/service';
 describes.sandboxed('StandardActions', {}, () => {
   let standardActions;
   let mutateElementStub;
+  let deferMutateStub;
 
-  beforeEach(() => {
-    standardActions = new StandardActions(new AmpDocSingle(window));
-    mutateElementStub = sandbox.stub(
+  function createAmpElement() {
+    const element = document.createElement('div');
+    element.classList.add('i-amphtml-element');
+    return element;
+  }
+
+  function stubMutate(methodName) {
+    return sandbox.stub(
         standardActions.resources_,
-        'mutateElement',
+        methodName,
         (unusedElement, mutator) => mutator());
+  }
+
+  function expectElementToHaveBeenHidden(element) {
+    expect(mutateElementStub).to.be.calledOnce;
+    expect(mutateElementStub.firstCall.args[0]).to.equal(element);
+    expect(element.style.display).to.equal('none');
+  }
+
+  function expectElementToHaveBeenShown(element) {
+    expect(deferMutateStub).to.be.calledOnce;
+    expect(deferMutateStub.firstCall.args[0]).to.equal(element);
+    expect(element.style.display).to.not.equal('none');
+  }
+
+  beforeEach(() => {
+    standardActions = new StandardActions(new AmpDocSingle(window));
+    mutateElementStub = stubMutate('mutateElement');
+    deferMutateStub = stubMutate('deferMutate');
   });
 
   describe('"hide" action', () => {
     it('should handle normal element', () => {
       const element = document.createElement('div');
       standardActions.handleHide({target: element});
+      // expectElementToHaveBeenHidden(element);
+    });
+
+    it('should handle AmpElement', () => {
+      const element = createAmpElement();
+      element.collapse = sandbox.stub();
+
+      standardActions.handleHide({target: element});
       expect(mutateElementStub).to.be.calledOnce;
       expect(mutateElementStub.firstCall.args[0]).to.equal(element);
-      expect(element.style.display).to.equal('none');
+      expect(element.collapse).to.be.calledOnce;
+    });
+  });
+
+  describe('"show" action', () => {
+    it('should handle normal element', () => {
+      const element = document.createElement('div');
+      standardActions.handleShow({target: element});
+      expectElementToHaveBeenShown(element);
     });
 
     it('should handle AmpElement', () => {
+      const element = createAmpElement();
+      element.expand = sandbox.stub();
+
+      standardActions.handleShow({target: element});
+      expect(deferMutateStub).to.be.calledOnce;
+      expect(deferMutateStub.firstCall.args[0]).to.equal(element);
+      expect(element.expand).to.be.calledOnce;
+    });
+
+  });
+
+  describe('"toggle" action', () => {
```

Comment:
and a test here to handle the comment above about computedStyle. Something along the lines of creating a `style` tag with a class like `.hide { display: none }` and using that class instead of `display=none`.